cdap-master/src/main/java/co/cask/cdap/data/tools/HBaseQueueDebugger.java

```diff
@@ -499,6 +500,7 @@ private static Injector createInjector(boolean disableAuthorization) throws Exce
       new ProgramRunnerRuntimeModule().getDistributedModules(),
       new SystemDatasetRuntimeModule().getDistributedModules(),
       new NotificationServiceRuntimeModule().getDistributedModules(),
+      new LoggingModules().getDistributedModules(),
```

Comment:
How does it work before?

Reply:
The bindings for the RemoteExecutionTwillRunnerService class did not previously have any dependencies on any logging bindings, but now it does (due to RuntimeMonitor depending on logging modules' dependencies).

See the failing test cases here: https://builds.cask.co/browse/CDAP-DUT6503-4/test